providers/common/include/prov/providercommonerr.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Generated by util/mkerr.pl DO NOT EDIT
- * Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2020-2021 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Is there any purpose for this file now?

Reply:
It declares `err_load_PROV_strings_int`, which we want to keep private